conf/nomad/mc.hcl

```diff
@@ -37,7 +37,7 @@ job "${service_name}" {
       }
       template {
         // todo: put under `local/secrets`
-        destination = "${data_dir}/.envs"
+        destination = "local/data/.envs"
```

Comment:
Why change back? 🤔

Reply:
didnt work with ${data_dir} :man_shrugging:

Reply:
What was the error?

Reply:
Why do you need it `data_dir` variable under `template` stanza.

These directories are [predefined by nomad](https://www.nomadproject.io/docs/runtime/environment#task-directories)

Reply:
Do not mix the context of destination folders. Under template stanza, there are 3 top-level predefined directories available. These directories use for concrete allocation (we use docker driver, so allocation is docker container). These folders are for the container itself, but not for application which runs inside the container

Reply:
<img width="1037" alt="Screenshot 2020-11-03 at 12 30 48" src="https://user-images.githubusercontent.com/15572799/97980172-a6933380-1dd0-11eb-8b40-7bbd09a94393.png">
<img width="911" alt="Screenshot 2020-11-03 at 12 30 57" src="https://user-images.githubusercontent.com/15572799/97980174-a7c46080-1dd0-11eb-841b-1dce9f520249.png">


Examples

Reply:
Examples under `secrets/`
<img width="1083" alt="Screenshot 2020-11-03 at 12 33 38" src="https://user-images.githubusercontent.com/15572799/97980364-f07c1980-1dd0-11eb-9d76-338f2c27cf59.png">
<img width="1064" alt="Screenshot 2020-11-03 at 12 33 47" src="https://user-images.githubusercontent.com/15572799/97980366-f245dd00-1dd0-11eb-9400-f38f7f540445.png">

Reply:
I would suggest using `secrets/.env` as a destination in this example, due to user-provided credentials setup via env variables